src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,55 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  searchInputValue: string;
+  setSearchInputValue: (todos: string) => void;
+  selectedOption: string;
+  setSelectedOption: (option: string) => void;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  searchInputValue,
+  setSearchInputValue,
+  selectedOption,
+  setSelectedOption,
+}) => {
```

Comment:
It seems you're directly interacting with the DOM in the onChange handlers (line 20 and line 37). While this isn't a direct DOM manipulation, it's generally recommended to handle events the 'React way' by creating specific methods for them. Consider following the naming conventions for event handlers, e.g., 'handleSelectChange' and 'handleInputChange'.